src/main/drivers/pwm_output.h

```diff
@@ -165,10 +167,18 @@ void servoDevInit(const servoDevConfig_t *servoDevConfig);
 void pwmServoConfig(const struct timerHardware_s *timerHardware, uint8_t servoIndex, uint16_t servoPwmRate, uint16_t servoCenterPulse);
 
 #ifdef USE_DSHOT
+typedef uint8_t(*loadDmaBufferFuncPtr)(motorDmaOutput_t *const motor, uint16_t packet);  // function pointer used to encode a digital motor value into the DMA buffer representation
+
+uint16_t prepareDshotPacket(motorDmaOutput_t *const motor, float value);
+
+extern loadDmaBufferFuncPtr loadDmaBufferPtr;
+
+uint8_t loadDmaBufferDshot(motorDmaOutput_t *const motor, uint16_t packet);
```

Comment:
loadDmaBuffer* is used only in pwm_output.c, no need to publish it. It may be  static too.